src/dodal/beamlines/i22.py

```diff
@@ -30,8 +34,50 @@ def it(
     return device_instantiation(
         TetrammDetector,
         "it",
-        "-EA-TTRM-02",
+        "-EA-TTRM-02:",
+        wait_for_connection,
+        fake_with_ophyd_sim,
+        directory_provider=get_directory_provider(),
+    )
+
+
+def oav_driver(
+    wait_for_connection: bool = True, fake_with_ophyd_sim: bool = False
+) -> ADAravisDriver:
+    return device_instantiation(
+        ADAravisDriver,
+        "oav_driver",
```

Comment:
I believe these names are required due to device_instantiation device tree walking?